lib/constructs/identity-provider.ts

```diff
@@ -107,7 +107,7 @@ export class CognitoIdentityProviderClient extends Construct {
 
   constructor(scope: Construct, id: string, props: CognitoIdentityProviderClientProps) {
     super(scope, id);
-    const client = props.userPool.addClient("Client", {
+    const client = props.userPool.addClient(id, {
```

Comment:
This change is to make it so an IdP can actually have more than one client. I think @zachclark-ccpo mentioned before that something like this may be necessary and I said that was incorrect. _I_ was actually the one who was incorrect.